client/reader/components/quick-post/index.tsx

```diff
@@ -51,7 +51,11 @@ function QuickPost( {
 	const translate = useTranslate();
 	const locale = useLocale();
 	const recordReaderTracksEvent = useRecordReaderTracksEvent();
-	const [ postContent, setPostContent ] = useState( '' );
+	const STORAGE_KEY = 'reader_quick_post_content';
+	const [ postContent, setPostContent ] = useState( () => {
+		// Use localStorage to save content between sessions.
+		return localStorage.getItem( STORAGE_KEY ) || '';
```

Comment:
Should we sanitize this data somehow? 🤔 
Seems like the input sanitization should already be happening on the backend (while saving the post) or by the editor when it evaluates the content. Can take a closer look tomorrow.

Reply:
I think this should be safe as is since it should be sanitized server-side, and we're not injecting it anywhere else.